tests/test_routes.py

```diff
@@ -0,0 +1,132 @@
+from app.models.planet import Planet
+
+
+def test_get_all_planets_returns_empty_list_when_db_is_empty(client):
+    #act
+    response = client.get("/planets")
+
+    #assert
+    assert response.status_code == 200
+    assert response.get_json() == []
+
+def test_get_one_planet_by_id(client, multiple_planets):
+    # Act
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == {
+        "description": "A dusty, cold, desert world with a very thin atmosphere",
+        "gravity": "3.721 m/s²",
+        "habitable": False,
+        "id": 1,
+        "moon_of_planet": "Phobos & Deimos",
+        "name": "Mars",
+        "nickname": "Red Planet",
+        "size": "6,792 km"
+    }
+
+def test_get_one_planet_with_no_data(client):
+    # Act
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 404
+    assert response_body == {"message": "Planet 1 not found"}
+
+def test_get_one_planet_with_id_not_found(client, multiple_planets):
+    # Act
+    response = client.get("/planets/5")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 404
+    assert response_body == {"message": "Planet 5 not found"}
+
+def test_get_list_of_all_planets(client, multiple_planets):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == [{
+        "description": "A dusty, cold, desert world with a very thin atmosphere",
+        "gravity": "3.721 m/s²",
+        "habitable": False,
+        "id": 1,
+        "moon_of_planet": "Phobos & Deimos",
+        "name": "Mars",
+        "nickname": "Red Planet",
+        "size": "6,792 km"
+    }, {
+        "name": "Saturn",
+        "description": "The second-largest planet in solar system.",
+        "size": "36,183.7 miles",
+        "moon_of_planet": "Iapetus, Rhea, Dione, and Tethys",
+        "habitable": False,
+        "id": 2,
+        "gravity": "10.44 m/s²",
+        "nickname": "Ringed Planet"
+    }, {
+        "name": "Neptune",
+        "description": "Dark, cold, and whipped by supersonic winds, ice giant.",
+        "size": "24,622 km",
+        "moon_of_planet": "Triton",
+        "habitable": False,
+        "id": 3,
+        "gravity": "11.15 m/s²",
+        "nickname": "Ice Giant"
+    }]
```

Comment:
Similar to my comment above for test_get_one_planet_by_id, if the fixture `multiple_planets` returned a list of saved planets, you can check the response_body against the saved planets. 

```python 
# incomplete assertions, but some examples for what you can write so that the test is more robust
    assert response_body[0]["id"] == multiple_planets[0].id
    assert response_body[0]["name"] == multiple_planets[0].name
    assert response_body[0]["description"] == multiple_planets[0].description
    assert response_body[0]["habitable"] == multiple_planets[0].habitable
    assert response_body[1]["id"] == multiple_planets[1].id
    assert response_body[1]["name"] == multiple_planets[1].name
    assert response_body[1]["description"] == multiple_planets[1].description
    assert response_body[1]["habitable"] == multiple_planets[1].habitable

# finish the rest of the assertions for the 3rd planet